{
  "Header": {
    "Original": "CVE-2020-25815 (An issue was discovered in MediaWiki 1.32.x through 1.34.x before 1.34 ...)",
    "Line": 199029,
    "ID": "CVE-2020-25815",
    "Description": "(An issue was discovered in MediaWiki 1.32.x through 1.34.x before 1.34 ...)"
  },
  "Annotations": [
    {
      "Original": "- mediawiki 1:1.35.0-1",
      "Line": 199030,
      "Type": "package",
      "Package": "mediawiki",
      "Kind": "fixed",
      "Version": "1:1.35.0-1"
    },
    {
      "Original": "[buster] - mediawiki \u003cnot-affected\u003e (Vulnerable code introduced in 1.32)",
      "Line": 199031,
      "Type": "package",
      "Release": "buster",
      "Package": "mediawiki",
      "Kind": "not-affected",
      "Description": "Vulnerable code introduced in 1.32"
    },
    {
      "Original": "[stretch] - mediawiki \u003cnot-affected\u003e (Vulnerable code introduced later)",
      "Line": 199032,
      "Type": "package",
      "Release": "stretch",
      "Package": "mediawiki",
      "Kind": "not-affected",
      "Description": "Vulnerable code introduced later"
    },
    {
      "Original": "NOTE: https://lists.wikimedia.org/pipermail/wikitech-l/2020-September/093888.html",
      "Line": 199033,
      "Type": "NOTE",
      "Description": "https://lists.wikimedia.org/pipermail/wikitech-l/2020-September/093888.html"
    },
    {
      "Original": "NOTE: https://phabricator.wikimedia.org/T256171",
      "Line": 199034,
      "Type": "NOTE",
      "Description": "https://phabricator.wikimedia.org/T256171"
    }
  ]
}